title-1-commodities/dmc_subprograms_data.json

```diff
@@ -1,11 +1,6 @@
 {
-  "programs": [
-    {
-      "programName": "Dairy Margin Coverage, Subtitle D",
-      "paymentInDollars": 1969315257.26,
-      "totalCounts": 1259532,
-      "averageRecipientCount": 157441.5,
-      "subPrograms": []
-    }
-  ]
+  "subtitleName": "Dairy Margin Coverage, Subtitle D",
+  "subtitlePaymentInDollars": 1969315257.26,
+  "totalCounts": 1259532,
+  "averageRecipientCount": 157441.5
```

Comment:
Here, if needed, `"programs": []` could be added for consistency. Other details look good to me. I will approve. Thanks.

Reply:
add "programs": []